packages/template-ui/src/components/ChannelFooter.vue

```diff
@@ -34,6 +34,14 @@ export default {
       return EndlessLogo;
     },
   },
+  $trs: {
+    bestOfTheWeb: {
+      // FIXME: This isn’t translatable to languages where the logo can’t be a suffix,
+      // or RTL languages.
+      message: '<span class="font-weight-bold">Best of the Web</span> curated by ',
```

Comment:
```suggestion
      message: '<strong>Best of the Web</strong> curated by ',
```

While we're at it, could we use `<strong>` here instead of a `<span …>`? (And it'll probably be styled the same, but double check that). It'll be less trouble for translating.